### PR TITLE
describe what happens when the PTO timer expires

### PR DESCRIPTION
**Pull request by marten-seemann**

### CODE REVIEW

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -506,9 +506,10 @@ delay sending an acknowledgement.
 The PTO value MUST be set to at least kGranularity, to avoid the timer expiring
 immediately.
 
-When a PTO timer expires, the PTO period MUST be set to twice its current value.
-This exponential reduction in the sender's rate is important because the PTOs
-might be caused by loss of packets or acknowledgements due to severe congestion.
+When a PTO timer expires, the sender sends a probe packet as described in the
+next section. The PTO period MUST be set to twice its current value. This
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md]:
```suggestion
next section. The PTO period then MUST be set to twice its current value. This
```

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -506,9 +506,10 @@ delay sending an acknowledgement.
 The PTO value MUST be set to at least kGranularity, to avoid the timer expiring
 immediately.
 
-When a PTO timer expires, the PTO period MUST be set to twice its current value.
-This exponential reduction in the sender's rate is important because the PTOs
-might be caused by loss of packets or acknowledgements due to severe congestion.
+When a PTO timer expires, the sender sends a probe packet as described in the
```
Comment (ianswett) [inline draft-ietf-quic-recovery.md]: It could be two probe packets, so maybe "sends one or two probe packets"?

Comment (nibanks) [reply draft-ietf-quic-recovery.md]: Or simply "probes the network"?

Comment (martinthomson) [reply draft-ietf-quic-recovery.md]: Or simply nothing.  A probe packet is sent if two packets are sent, and this is not normative text.

Comment (marten-seemann) [reply draft-ietf-quic-recovery.md]: I like @nibanks's suggestion. It's independent of how the probing is actually done, which we describe in detail later.

File: draft-ietf-quic-recovery.md [hunk 3]
```diff
@@ -506,9 +506,10 @@ delay sending an acknowledgement.
 The PTO value MUST be set to at least kGranularity, to avoid the timer expiring
 immediately.
 
-When a PTO timer expires, the PTO period MUST be set to twice its current value.
-This exponential reduction in the sender's rate is important because the PTOs
-might be caused by loss of packets or acknowledgements due to severe congestion.
+When a PTO timer expires, the sender probes the network as described in the next
+section. The PTO period MUST be set to twice its current value. This exponential
+reduction in the sender's rate is important because the PTOs might be caused by
```
Comment (mikkelfj) [inline draft-ietf-quic-recovery.md]:
```suggestion
backoff in the sender's rate is important because the PTOs might be caused by
```

Comment (marten-seemann) [reply draft-ietf-quic-recovery.md]: Out of scope for this PR.

**Pull request merged**